modules/server_processes_manager/test/server_processes_managerTest.php

```diff
@@ -134,14 +134,19 @@ function testFilters()
     {
         $this->markTestSkipped("Skipping long test");
         return;
-        $this->_testFilter("/server_processes_manager/", "pid", "317", "1 rows");
-        $this->_testFilter("/server_processes_manager/", "type", "mri_upload", "51");
-        $this->_testFilter(
-            "/server_processes_manager/",
-            "userid",
-            "admin",
-            "51"
-        );
+        //this->_testFilter("/server_processes_manager/", "pid", "317", "1 rows");
+        //this->_testFilter(
+        //"/server_processes_manager/",
+        //"type",
+        //"mri_upload",
+        //"51"
+        //);
+        //this->_testFilter(
+        //   "/server_processes_manager/",
+        //   "userid",
```

Comment:
Why is this code all commented out? If it's deleted, it should be deleted (same with the other test above)

Reply:
Works for me, just thought it might have been left in as a reference.